forward_engineering/helpers/updateHelpers/viewHelper.js

```diff
@@ -1,17 +1,11 @@
 const { getViewScript: generateViewScript, getOptionsScript } = require('../viewHelper');
 const { tab } = require('../generalHelper');
 const { dependencies } = require('../appDependencies');
+const { AlterScriptDto } = require("../types/AlterScriptDto");
 let _;
```

Comment:
Find a way to get rid of caching _ dependency. Either pass it as functions' dependencies or use globall dependency object

Reply:
I propose not to modify this, as we expecting the update of the flow how we're working with plugin dependencies

Reply:
got lodash from the global dependencies object